### PR TITLE
update README for luadata fixed size getter/setters

### PR DESCRIPTION
**Pull request by sheharyaar**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -888,6 +888,36 @@ _data.setstring()_ insert the string `s`
 into the memory referenced by a `data` object and a byte `offset`,
 starting from zero.
 
+#### `data.get(u)int(8/16/32/64)(d, offset), d:get(u)int(8/16/32/64)(offset)`
```
Comment (lneto) [inline README.md:891]: I would have one entry/subsection per API for indexing it properly..

**Comment by lneto** [comment]
> Irrelevant here, but by the way, the README isn't a bit too big ? Will make sense to move most of it to a dedicated docs/, and reorganize it a bit ...

I think so.. perhaps moving sections to subdocuments.. I would really appreciate this sort of reorganization.. perhaps using LDocs..

**Comment by sheharyaar** [comment]
Should I create seperate subsections for now ?

**Comment by lneto** [comment]
> Should I create seperate subsections for now ?

It's up to actually.. I would not spend much time on this now and leave it for a separate issue..

**Review by lneto — APPROVED** [review]

**Pull request merged**